ibis/expr/operations/udfs.py

```diff
@@ -0,0 +1,109 @@
+from __future__ import annotations
+
+import abc
+import functools
+import inspect
+import typing
+from typing import Any
+
+from public import public
+
+import ibis.expr.datatypes as dt
+import ibis.expr.operations as ops
+import ibis.expr.rules as rlz
+from ibis.common.annotations import attribute
+
+
+@public
+class ScalarUDFNode(ops.Value):
```

Comment:
Can you remove the `Node` postfix?

Reply:
What does `ScalarUDF` mean?

We have another nomenclature `ElementWise` in the `ibis/expr/operations/vectorized.py` file. It would be nice to unify the operations over there and here.